test/EndToEnd/tests/BuildIntegratedTest.ps1

```diff
@@ -111,6 +111,7 @@ function Test-BuildIntegratedUninstallNonExistantPackage {
 }
 
 function Test-BuildIntegratedLockFileIsCreatedOnBuild {
+    [SkipTest('https://github.com/NuGet/Home/issues/12189')]
```

Comment:
This doesn't work with passing the context param. 

If you look at the test tab, you'll see that the test actually ran.

Reply:
I forgot it, addressed.